code/style.css

```diff
@@ -1,58 +1,140 @@
+* { 
+  box-sizing: border-box;
+ }
+ *:before,
+ *:after {
+  box-sizing: border-box;
+ }
+
+html,
+body {
+ height: 100%;
+ position: relative;
+}
+
 body {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  font-family: GabrielWeissFriendsFont;
+  background-image: url(./assets/background.jpg);
+  background-size: cover;
 }
 
-p {
-  margin: 1px;
-  font-weight: 300;
-  background-color: #fff;
+.main-container {
+  min-height: 100vh; 
+  overflow: hidden;
+  display: block;
+  position: relative;
+  padding-bottom: 50px; 
+}
+
+
+h1 {
+  font-size: 80px;
+  margin-top: 10px;
+  margin-bottom: 0;
+  font-family: GabrielWeissFriendsFont;
+  color: #000000;
+  text-align: center;
+}
+
+h2 {
+  margin-top: 0;
+  font-family: GabrielWeissFriendsFont;
+  color: #000000;
+  text-align: center;
+  font-size: 30px;
 }
 
 .cards {
-  margin: 50px auto;
-  width: 90%;
-  height: 600px;
-  background: #ccc;
+  margin: 30px auto 50px auto;
+  width: 100%;
+  height: 500px;
   display: flex;
   justify-content: space-around;
   align-items: center;
+  background-color: rgba(255, 255, 255, 0.3);
+  backdrop-filter: blur(3px);
+  -webkit-backdrop-filter: blur(3px);
 }
 
-.killer-deck {
-  width: 250px;
-  height: 400px;
-  background: #000;
-  text-align: center;
-  padding-top: 30px;
-  border-radius: 16px;
+.card-name {
+  font-family: GabrielWeissFriendsFont;
+  font-size: 25px;
 }
 
-.weapon-deck {
-  width: 250px;
-  height: 400px;
-  background: rgb(10, 53, 10);
-  text-align: center;
-  padding-top: 30px;
-  border-radius: 16px;
+p {
+  margin: 2px;
+  font-weight: 300;
+  font-size: 15px;
 }
 
-.room-deck {
-  width: 250px;
-  height: 400px;
-  background: midnightblue;
+#friendsImage, #foodsImage, #roomsImage {
```

Comment:
I don't know if this is necessary with the # but not sure, correct me if Im wrong please! 😁

Reply:
I have used because I did not want to do one more different class for the images!(:

Reply:
Honestly, you should Lili :) In CSS we should use only classes, no id's